include/nestegg/nestegg.h

```diff
@@ -387,6 +387,13 @@ int nestegg_read_last_packet(nestegg * context,
                              unsigned int track,
                              nestegg_packet ** packet);
 
+/** Read total frames count for a track without affecting current parser state.
```

Comment:
We should make it clear that this should be called before any call to `nestegg_read_packet`.  The `ancestor` assert in `ne_read_cluster_frames_count` verifies this.

nit: "frame count"